src/Raven.Studio/typescript/common/pwaInstaller.ts

```diff
@@ -0,0 +1,70 @@
+﻿/// <reference path="../../typings/tsd.d.ts" />
+
+/**
+ * Service that aids in performing installation for progressive web apps (PWAs).
+ * For more info, see https://developers.google.com/web/fundamentals/app-install-banners
+ * */
+class pwaInstaller {
+    
+    private deferredInstallPrompt: BeforeInstallPromptEvent | null = null;
```

Comment:
we don't use explicit null notation in current version of studio. Let's be consistent.

Reply:
:-(